4.sc_Morphology_Neural_Network_MLP_Model/0.model_training/scripts/Hyperparameter_Optimization_model_binary.py

```diff
@@ -70,27 +70,33 @@
 from utils.utils import df_stats
 
 # %%
-# Import Data
-# set data file path under pathlib path for multi-system use
-
-
-file_path = Path(
-    "../../../../Extracted_Features_(CSV_files)/interstellar_wave3_sc_norm_fs_cellprofiler.csv.gz"
-)
-
-file_path = Path(
-    "../../../../Extracted_Features_(CSV_files)/SHSY5Y_preprocessed_df_sc_norm.parquet"
-)
-
-df = pq.read_table(file_path).to_pandas()
-# df = pd.read_csv(file_path, engine="pyarrow")
+# Parameters
+CELL_TYPE = "PBMC"
+CONTROL_NAME = "DMSO_0.100_DMSO_0.025"
+TREATMENT_NAME = "Thapsigargin_1.000_DMSO_0.025"
+MODEL_NAME = "DMSO_0.025_vs_Thapsigargin_1"
 
 # %%
-data = Path("../../MLP_utils/config.toml")
+data = Path("../../MLP_utils/binary_config.toml")
```

Comment:
Consider using `pathlib.Path.resolve(strict=True)` as this will raise an error if the file is not found and will also return the absolute path of the file making it easier to process.

Reply:
Also consider changing the name `data` to something more accurate. It seems what you're loading are machine learning configs. I suggest using `ml_configs` might be a better name? we want to let the reader know what we're loading in.

Reply:
addressed comment, added both suggestions